rust/agama-cli/doc/backend-for-testing.md

```diff
@@ -24,48 +24,42 @@ bit old and the D-Bus API was mismatched between frontend and backend.
 
 ## Details
 
-- Frontend: [d-installer-cli][], this repo
-- Backend:  [d-installer][]
-
-[d-installer-cli]: https://github.com/yast/d-installer-cli
-[d-installer]: https://github.com/yast/d-installer
-
 The container used is built in
-[OBS YaST:Head:Containers/d-installer-testing](d-installer-testing) and
+[OBS YaST:Head:Containers/agama-testing](agama-testing) and
 downloaded from registry.o.o specified below.
 
-[d-installer-testing]: https://build.opensuse.org/package/show/YaST:Head:Containers/d-installer-testing
+[agama-testing]: https://build.opensuse.org/package/show/YaST:Head:Containers/agama-testing
 
 I basically picked the useful bits from the `integration-tests` part
-of [d-installer/.../ci.yml][ci.yml].
+of [.github/workflows/ci.yml][ci.yml].
 
-[ci.yml]: https://github.com/yast/d-installer/blob/25462f57ab695d6910beb59ff0b21a7afaeda47e/.github/workflows/ci.yml
+[ci.yml]: https://github.com/openSUSE/agama/blob/25462f57ab695d6910beb59ff0b21a7afaeda47e/.github/workflows/ci.yml
```

Comment:
maybe we should reference here master branch instead?

Reply:
No, that's intentional. I want the reader to see the file as I saw it even if the branch evolves. You can easily go to its tip, but not the other way around, guessing what I saw.